app/repositories/ApprovalProcessReviewRepository.scala

```diff
@@ -89,4 +91,31 @@ class ApprovalProcessReviewRepositoryImpl @Inject() (implicit mongoComponent: Re
     //$COVERAGE-ON$
   }
 
+  def updatePageReview(id: String, version: Int, pageUrl: String, reviewInfo: ApprovalProcessPageReview): Future[RequestOutcome[Unit]] = {
+
+    val selector = Json.obj("ocelotId" -> id, "version" -> version, "pages.pageUrl" -> pageUrl)
+    val modifier =
+      Json.obj(
+        "$set" -> Json.obj(
+          "pages.$.result" -> reviewInfo.result,
+          "pages.$.status" -> reviewInfo.status,
+          "pages.$.comment" -> reviewInfo.comment,
+          "pages.$.updateUser" -> reviewInfo.updateUser,
+          "pages.$.updateDate" -> Json.obj("$date" -> LocalDateTime.now().toInstant(ZoneOffset.UTC).getEpochSecond * 1000)
```

Comment:
LocalDateTime.now().atZone(ZoneId.of("UTC")).toInstant().toEpochMilli()

Reply:
I knew there must be a way to get millis, I didn't look hard enough :) thanks

Reply:
Done - in next commit